<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMainWindow</class>
 <widget class="QMainWindow" name="QMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>715</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="hl1">
      <item>
       <widget class="QPushButton" name="btn2_update">
        <property name="text">
         <string>ویرایش مشخصات</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn3_insert">
        <property name="text">
         <string>کاربر جدید</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn1_select">
        <property name="text">
         <string>نمایش مشخصات</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>715</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
